Language_Parser/actionVerbs.py

```diff
@@ -34,11 +34,18 @@ def examine(info):
         print(errorString)
         return
 
+    game = info["Game"]
+    player = game.getPlayer()
+    room = player.getLocation()
+    allItems = player.getInventory() + room.getAccessibleItems()
```

Comment:
allItems is being created twice, once here and again in lines 51-53.

Reply:
Thanks, fixed it.